### PR TITLE
Fix DB upgrades of execution_flows

### PR DESCRIPTION
**Pull request by juhoautio**
Fixing / cleaning up after these PRs:
- https://github.com/azkaban/azkaban/pull/2688/files#r699318660
- https://github.com/azkaban/azkaban/pull/2779/files#r699317153

### CODE REVIEW

File: azkaban-db/src/main/sql/updgrade.2ab81c191047f14a77adbfe8eda3bcaa56974cfd.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+-- TODO what are the from & to versions for this?
+-- the original change of create.execution_flows.sql was done in commit:
+-- 2ab81c191047f14a77adbfe8eda3bcaa56974cfd
+
+-- DB Migration from release ??? to ???
```
Comment (juhoautio) [inline azkaban-db/src/main/sql/updgrade.2ab81c191047f14a77adbfe8eda3bcaa56974cfd.sql:5]: Could someone help me figure out the answer to this? The actual release that this affects is the one that included https://github.com/azkaban/azkaban/pull/2779.

Comment (amielkelinkedin) [reply azkaban-db/src/main/sql/updgrade.2ab81c191047f14a77adbfe8eda3bcaa56974cfd.sql:5]: @aditya1105, Can you answer this?

Comment (juhoautio) [reply azkaban-db/src/main/sql/updgrade.2ab81c191047f14a77adbfe8eda3bcaa56974cfd.sql:5]: @amielkelinkedin thanks for commenting..

I found that commit 2ab81c1 (of https://github.com/azkaban/azkaban/pull/2779) is listed in https://github.com/azkaban/azkaban/releases/tag/4.0.0
> 2ab81c1 Added mechanism to rampup Containerized from POLL dispatch

The previous release tag is 3.90.0.

So I will mark this file `DB Migration from release 3.90.0 to 4.0.0`.

Comment (juhoautio) [reply azkaban-db/src/main/sql/updgrade.2ab81c191047f14a77adbfe8eda3bcaa56974cfd.sql:5]: Now I'm confused though, how can it be that the repo contains this file: upgrade.4.10.0.to.4.11.0.sql (created by @ypadron-in) – even though there are no such release tags in git. Is this some LinkedIn internal versioning?

**Comment by juhoautio** [comment]
This should be now all good to be merged as far as I can see.

**Comment by juhoautio** [comment]
@djaiswal83 @aditya1105.. anyone?

**Review by jakhani — APPROVED** [review]

**Comment by juhoautio** [comment]
@jakhani thanks ^ -- could you merge this if ok? I don't have permission to merge myself.

**Pull request merged**